mediagalaxy.ro,"HTTPSConnectionPool(host='mediagalaxy.ro', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mediagalaxy.ro', port=80): Read timed out. (read timeout=2)"
mediclinic.ae,"HTTPSConnectionPool(host='mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
medvestnik.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
medwave.cl,"HTTPSConnectionPool(host='medwave.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80080fd190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='medwave.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80080f3490>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mefos.hr,"HTTPSConnectionPool(host='mefos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800aa5c2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mefos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80090541f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
megashop.hr,"HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)"
meiji.ac.jp,"HTTPSConnectionPool(host='meiji.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800841e190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meiji.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800998b760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mekshat.com,"HTTPSConnectionPool(host='mekshat.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1124)')))",Exceeded 30 redirects.
melody4arab.xyz,"HTTPSConnectionPool(host='melody4arab.xyz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8008a8cac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='melody4arab.xyz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8009b5ddf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melonbooks.co.jp,"HTTPSConnectionPool(host='melonbooks.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: EE certificate key too weak (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.melonbooks.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
men.gov.ma,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mendoza.gov.ar,"HTTPSConnectionPool(host='mendoza.gov.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80085dd0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mendoza.gov.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8004e80040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menessaptieka.lv,"HTTPSConnectionPool(host='menessaptieka.lv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8004cb0040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menessaptieka.lv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8007960700>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menlhk.go.id,"HTTPSConnectionPool(host='menlhk.go.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8009a0eaf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menlhk.go.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80067f4910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menofia.edu.eg,"HTTPSConnectionPool(host='menofia.edu.eg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800a7cfe20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menofia.edu.eg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8008b3d340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menoinfo.fi,"HTTPSConnectionPool(host='menoinfo.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8004ab9640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menoinfo.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800971dac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menpan.go.id,"HTTPSConnectionPool(host='menpan.go.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='menpan.go.id', port=443): Read timed out. (read timeout=2)"
menupages.ie,"HTTPSConnectionPool(host='menupages.ie', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8008186c70>, 'Connection to menupages.ie timed out. (connect timeout=2)'))","HTTPConnectionPool(host='menupages.ie', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f800970b910>, 'Connection to menupages.ie timed out. (connect timeout=2)'))"
mep.go.cr,Exceeded 30 redirects.,Exceeded 30 redirects.
mercadopublico.cl,"HTTPSConnectionPool(host='mercadopublico.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80093ece80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercadopublico.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80079da2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mercedes.gob.ar,"HTTPSConnectionPool(host='mercedes.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800ced36d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercedes.gob.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8006a22a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
merlin.pl,"HTTPSConnectionPool(host='merlin.pl', port=443): Read timed out.","HTTPSConnectionPool(host='merlin.pl', port=443): Read timed out."
meteo.it,"HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
meteo.lt,"HTTPSConnectionPool(host='meteo.lt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f80079e3b80>, 'Connection to meteo.lt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.meteo.lt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f800632cca0>, 'Connection to www.meteo.lt timed out. (connect timeout=2)'))"
meti.go.jp,"HTTPSConnectionPool(host='meti.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8003c9ba30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meti.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80067add30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metjekutindevut.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
metro.tokyo.jp,"HTTPSConnectionPool(host='metro.tokyo.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80091c70d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='metro.tokyo.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80092759a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metrouniv.ac.id,"HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)"
mext.go.jp,"HTTPSConnectionPool(host='mext.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800adca250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mext.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8008988fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.ee,"HTTPSConnectionPool(host='mfa.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8009922970>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mfa.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800929aa30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.gov.sg,"HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))","HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))"
mfa.gov.tr,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPSConnectionPool(host='mfa.gov.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mhlw.go.jp,"HTTPSConnectionPool(host='mhlw.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8004105160>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mhlw.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800300b250>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
michoacan.gob.mx,"HTTPSConnectionPool(host='michoacan.gob.mx', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='michoacan.gob.mx', port=443): Read timed out. (read timeout=2)"
mides.gub.uy,"HTTPSConnectionPool(host='mides.gub.uy', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8009dd2340>, 'Connection to mides.gub.uy timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mides.gub.uy', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8004dc8e20>, 'Connection to mides.gub.uy timed out. (connect timeout=2)'))"
mie.lg.jp,"HTTPSConnectionPool(host='mie.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8008988370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mie.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8006f96160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
migdalor-news.co.il,"HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)"
mikmak.co.il,"HTTPSConnectionPool(host='mikmak.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8008744be0>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mikmak.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f80062d9430>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))"
mikronis.hr,"HTTPSConnectionPool(host='mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
milejemplos.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
milog.co.il,"HTTPSConnectionPool(host='milog.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='milog.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
min-mave.dk,"HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)"
min-saude.pt,"HTTPSConnectionPool(host='min-saude.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8009cfb580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='min-saude.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800943d4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minciencias.gov.co,"HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mindop.sk,"HTTPSConnectionPool(host='mindop.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='mindop.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8006286160>, 'Connection to mindop.sk timed out. (connect timeout=2)'))"
minec.gob.sv,"HTTPSConnectionPool(host='minec.gob.sv', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f80085b8a90>, 'Connection to minec.gob.sv timed out. (connect timeout=2)'))","HTTPConnectionPool(host='minec.gob.sv', port=80): Read timed out. (read timeout=2)"
minec.gob.ve,"HTTPSConnectionPool(host='minec.gob.ve', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800b60f760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minec.gob.ve', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800888aa60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineco.gob.gt,"HTTPSConnectionPool(host='mineco.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8001a748e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mineco.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80031d3dc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mined.gob.sv,"HTTPSConnectionPool(host='mined.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800ae84310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mined.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8009d46dc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minedu.gob.bo,"HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
minedu.sk,"HTTPSConnectionPool(host='minedu.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800669adf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minedu.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800ac7ffa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineducacion.gov.co,"HTTPSConnectionPool(host='mineducacion.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800ac7fdc0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mineducacion.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80017f8c10>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
minetilbud.dk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
minfin.bg,"HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)"
mininterior.gov.co,"HTTPSConnectionPool(host='mininterior.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f80094250d0>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mininterior.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f800af12910>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))"
ministeriodegobierno.gob.ec,"HTTPSConnectionPool(host='ministeriodegobierno.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ministeriodegobierno.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
ministeriodesalud.go.cr,"HTTPSConnectionPool(host='ministeriodesalud.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8009967af0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ministeriodesalud.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8006796b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minsa.gob.pe,"HTTPSConnectionPool(host='minsa.gob.pe', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800aadf550>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minsa.gob.pe', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8009ec8880>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintrabajo.gov.co,"HTTPSConnectionPool(host='mintrabajo.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8007818340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mintrabajo.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800983e3d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintransporte.gov.co,"HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minvu.cl,"HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minyu-net.com,"HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
mise.gov.it,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
missbloom.bg,"HTTPSConnectionPool(host='missbloom.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f80089ed100>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='missbloom.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f800b9ce580>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))"
missmundo.nl,"HTTPSConnectionPool(host='missmundo.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800378e130>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='missmundo.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8001b0e730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
misumi-ec.com,"HTTPSConnectionPool(host='misumi-ec.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800a0e1cd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='misumi-ec.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80093eca30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mizuhobank.co.jp,"HTTPSConnectionPool(host='mizuhobank.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8002169ac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mizuhobank.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8001b54d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mj.pt,"HTTPSConnectionPool(host='mj.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8005cdbdf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mj.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80015f8730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mju.ac.th,"HTTPSConnectionPool(host='mju.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f800a6aa2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mju.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80099a5b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mku.edu.tr,"HTTPSConnectionPool(host='mku.edu.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80016ddfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mku.edu.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80097ac9d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mladinska.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mlap.hu,"HTTPSConnectionPool(host='mlap.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mlap.hu' doesn't match 'dbmanager.hu'"")))","HTTPConnectionPool(host='mlap.hu', port=80): Read timed out. (read timeout=2)"
mlit.go.jp,"HTTPSConnectionPool(host='mlit.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8008c0f4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlit.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8008eb7070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mlsi.gov.cy,"HTTPSConnectionPool(host='mlsi.gov.cy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8001584f70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlsi.gov.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f800415da00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mm.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
moag.gov.il,"HTTPSConnectionPool(host='www.moag.gov.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.moag.gov.il', port=443): Read timed out. (read timeout=2)"
mobilepriceall.com,"HTTPSConnectionPool(host='www.mobilepriceall.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.mobilepriceall.com', port=80): Read timed out. (read timeout=2)"
mobilnisvet.com,"HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moda.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
modenaora.it,"HTTPSConnectionPool(host='modenaora.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='modenaora.it', port=80): Read timed out. (read timeout=2)"
moe.edu.sg,"HTTPSConnectionPool(host='moe.edu.sg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80016a40a0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='moe.edu.sg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8008ca8bb0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
moe.go.th,"HTTPSConnectionPool(host='moe.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8006781850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moe.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8001225760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moe.gov.jo,"HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)"
moe.gov.lk,"HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moe.gov.my,"HTTPSConnectionPool(host='moe.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.moe.gov.my', port=443): Read timed out. (read timeout=2)"
mof.gov.vn,"HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)"
mofa.go.jp,"HTTPSConnectionPool(host='mofa.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8001830b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mofa.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f80015f8ca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.bh,"HTTPSConnectionPool(host='moh.gov.bh', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f80080fd760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moh.gov.bh', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8008b4ea60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.jo,"HTTPSConnectionPool(host='moh.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='moh.gov.jo', port=80): Read timed out. (read timeout=2)"
moh.gov.my,"HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))","HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))"
moh.gov.vn,"HTTPSConnectionPool(host='moh.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='moh.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
mohap.gov.ae,"HTTPSConnectionPool(host='mohap.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mohap.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mohre.gov.ae,"HTTPSConnectionPool(host='mohre.gov.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mohre.gov.ae', port=443): Read timed out. (read timeout=2)"
moi.gov.ae,"HTTPSConnectionPool(host='moi.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='moi.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
